### ISSUE TITLE
[1.7.10] Server Tick Loop Error - TileEntity

### ISSUE THREAD

**Issue by bennyd87708** [BluSunrize/ImmersiveEngineering#3486]
### Description of the issue:
server crashes during startup

### Crashlog:
https://pastebin.com/gHLa40gL

### Versions & Modlist
IE Version 0.7.7
Forge 1.7.10 - 10.13.4.1614
modlist in crashlog (playing Infinity Evolved 1.7.10 3.0.2)

same error in https://github.com/BluSunrize/ImmersiveEngineering/issues/1555 but that thread is a joke and it was referenced in https://github.com/BluSunrize/ImmersiveEngineering/pull/1653 as being easily solvable, but I don't know how to compile it myself. Getting the error on a world that is very important to me, thanks for anything.

**Comment by BluSunrize** [comment]
Check Forge's config, they have an option to remove erroring TileEntities. Try that.

Beyond that, there's nothing we can do here. IE for 1.7 isn't supported anymore and hasn't been for well over 2 years.

**Comment by bennyd87708** [comment]
If anyone else stumbles across this error, enabling the forge option to remove erroring TileEntities did not do anything for me. I was able to get my world functional again my manually removing the tile entities for all wire connectors and multi-block structures from immersive engineering in MCEdit. I couldn't find a filter to find them all automatically so I found them out of pure luck remembering where they were all placed, as my crash report did not specify anything other than the dimension where the error occurred.
